e2e/tests-dfx/start.bash

```diff
@@ -92,6 +93,7 @@ teardown() {
 }
 
 @test "dfx restarts the replica" {
+  [[ "$USE_POCKETIC" ]] && skip "skipped for pocketic"
```

Comment:
can we have an equivalent test for dfx restarting pocketic?

Reply:
what do you think of a flag to write the PID when --background is used?

Reply:
I'd expect dfx to write a pid file unconditionally, like it does for the replica